src/Dfe.PlanTech.Web/Routing/GetRecommendationRouter.cs

```diff
@@ -42,6 +43,27 @@ public class GetRecommendationRouter(ISubmissionStatusProcessor router,
         };
     }
 
+    public async Task<IActionResult> GetRecommendationPreview(string sectionSlug,
+                                                              string recommendationSlug,
+                                                              string? maturity,
+                                                              RecommendationsController controller,
+                                                              CancellationToken cancellationToken)
+    {
+        await _router.GetJourneyStatusForSectionRecommendation(sectionSlug, cancellationToken);
+        var recommendation = await _getSubTopicRecommendationQuery.GetSubTopicRecommendation(_router.Section.Sys.Id, cancellationToken) ?? throw new ContentfulDataUnavailableException($"Could not find subtopic recommendation for:  {_router.Section.Name}");
+        var intro = recommendation.Intros.FirstOrDefault(intro => intro.Maturity == maturity) ?? recommendation.Intros.First();
```

Comment:
I don't think this is working for maturity because the url lowers the casing of maturity but the `intro.Maturity` starts with a capital letter

Reply:
Good shout, fixed.